src/store/wizardSlice.ts

```diff
@@ -46,6 +46,8 @@ export type ComplianceType = 'openscap' | 'compliance';
 
 export type UserWithAdditionalInfo = {
```

Comment:
If we're using a type intersection we can probably make the code a tiny bit more cleaner. This should work:

```typescript
export type UserWithAdditionalInfo = User & {
  confirmPassword: string;
};
```

Docs for type intersection: https://www.typescriptlang.org/docs/handbook/2/objects.html#intersection-types

Reply:
I use the NonNullable to ensure that a type doesnt allow null or undefined values. 
it makes all the properties of User to be required

Reply:
Ah, got it! :+1: